### PR TITLE
kernel: bump 6.1 to 6.1.93

### PR DESCRIPTION
**Pull request by zxlhhyccc**
Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.93

All other patches automatically rebased.

Build system: bcm47xx bcm53xx

### CODE REVIEW

File: target/linux/bcm27xx/patches-6.1/950-0081-smsx95xx-fix-crimes-against-truesize.patch [hunk 1]
```diff
@@ -25,21 +25,19 @@ Signed-off-by: Steve Glendinning <steve.glendinning@smsc.com>
...
  					  u32 *data)
  {
-@@ -1870,7 +1874,8 @@ static int smsc95xx_rx_fixup(struct usbn
+@@ -1875,6 +1879,8 @@ static int smsc95xx_rx_fixup(struct usbn
  				if (dev->net->features & NETIF_F_RXCSUM)
  					smsc95xx_rx_csum_offload(skb);
  				skb_trim(skb, skb->len - 4); /* remove fcs */
--				skb->truesize = size + sizeof(struct sk_buff);
 +				if (truesize_mode)
 +					skb->truesize = size + sizeof(struct sk_buff);
```
Comment (PolynomialDivision) [inline target/linux/bcm27xx/patches-6.1/950-0081-smsx95xx-fix-crimes-against-truesize.patch:33]: I just had a short look and this looks strange, that we add back the truesize although this commit also complains that truesize override should not be done at this stage:
https://github.com/torvalds/linux/commit/d50729f1d60bca822ef6d9c1a5fb28d486bd7593

Comment (zxlhhyccc) [reply target/linux/bcm27xx/patches-6.1/950-0081-smsx95xx-fix-crimes-against-truesize.patch:33]: @PolynomialDivision It makes sense, but it is temporarily retained due to no device testing. May I ask if your opinion no longer requires `truesize`？

Comment (PolynomialDivision) [reply target/linux/bcm27xx/patches-6.1/950-0081-smsx95xx-fix-crimes-against-truesize.patch:33]: Maybe @Noltari can say something about it? I don't understand why we even have this patch. It does not look like target is loading the driver with truesize_mode enabled? I would be in favor for removing this patch.

Comment (zxlhhyccc) [reply target/linux/bcm27xx/patches-6.1/950-0081-smsx95xx-fix-crimes-against-truesize.patch:33]: @Noltari Please give me your opinion.

Comment (hauke) [reply target/linux/bcm27xx/patches-6.1/950-0081-smsx95xx-fix-crimes-against-truesize.patch:33]: I think we can remove this patch from OpenWrt.
I do not have the hardware or know anything special about it.
The patch in OpenWrt deactivates setting `skb->truesize` by default, `truesize_mode` is `false` by default. 
The upstream kernel change removes setting it completely. 
https://github.com/gregkh/linux/commit/517e64bcc9634bb46f8fc6421e272b8da3d7d953

I do not see any code in OpenWrt setting the `truesize_mode` module parameter.

Comment (PolynomialDivision) [reply target/linux/bcm27xx/patches-6.1/950-0081-smsx95xx-fix-crimes-against-truesize.patch:33]: @zxlhhyccc Please remove the patch. I would remove it before doing the bump in a seperate commit.

Comment (zxlhhyccc) [reply target/linux/bcm27xx/patches-6.1/950-0081-smsx95xx-fix-crimes-against-truesize.patch:33]: @PolynomialDivision I will remove the patch and resubmit it later.

**Pull request closed without merging**